From a15543f7114b3f1ade8cce2cd1065e6aa60c14cc Mon Sep 17 00:00:00 2001
From: "Artyom V. Poptsov" <poptsov.artyom@gmail.com>
Date: Tue, 17 Sep 2019 22:31:50 +0300
Subject: [PATCH] configure.ac: Add missing AM_CONDITIONAL

* configure.ac: Add missing AM_CONDITIONAL for 'HAVE_LIBSSH_0_8' constant.
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 99d35af..1d8baf2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -69,6 +69,8 @@ PKG_CHECK_MODULES([LIBSSH_0_8], [libssh >= 0.8.0],
                                 [AC_DEFINE(HAVE_LIBSSH_0_8, 1, [Use libssh 0.8])],
                                 [AC_DEFINE(HAVE_LIBSSH_0_8, 0, [Use libssh < 0.8])])
 
+AM_CONDITIONAL(HAVE_LIBSSH_0_8, $HAVE_LIBSSH_0_8)
+
 PKG_CHECK_MODULES([LIBSSH_0_9], [libssh >= 0.9.0],
                                 [AC_DEFINE(HAVE_LIBSSH_0_9, 1, [Use libssh 0.9])],
                                 [AC_DEFINE(HAVE_LIBSSH_0_9, 0, [Use libssh < 0.9])])
